### ISSUE TITLE
[BUG] 输入 `hugo server -D`，会报错

### ISSUE THREAD

**Issue by sanbeichahegongheguo** [dillonzq/LoveIt#588]
### Describe the bug 描述你遇到的错误

输入 `hugo server -D`，会出现：

`\content\posts\index.zh-cn.md:1:1": timed out initializing value. This is most likely a circular loop in a shortcode`

这样的报错。请问是正常情况吗？

### Screenshots 屏幕截图

![图片](https://user-images.githubusercontent.com/32977199/109810018-f80adc80-7c63-11eb-8829-7b96c72feeac.png)

### Build Environment 构建环境

- OS:  Windows 10
- Theme version/commit [0.2.10, f787a4e5]
- Hugo version [v0.62.2-83E50184]

**Comment by khusika** [comment]
> ### Describe the bug 描述你遇到的错误
> 输入 `hugo server -D`，会出现：
> 
> `\content\posts\index.zh-cn.md:1:1": timed out initializing value. This is most likely a circular loop in a shortcode`
> 
> 这样的报错。请问是正常情况吗？
> 
> ### Screenshots 屏幕截图
> ![图片](https://user-images.githubusercontent.com/32977199/109810018-f80adc80-7c63-11eb-8829-7b96c72feeac.png)
> 
> ### Build Environment 构建环境
> * OS:  Windows 10
> * Theme version/commit [0.2.10, [f787a4e](https://github.com/dillonzq/LoveIt/commit/f787a4e5ad4edf60467658d10c286248dc5027a6)]
> * Hugo version [v0.62.2-83E50184]

At the moment, Hugo using deprecated Instagram linked API endpoint.I would like to make a PR for this issue but this repo is dead.
Read my article below this to fix those problem
[Instagram Shortcode in Hugo](https://khusika.com/instagram-shortcode-in-hugo/)

**Comment by sanbeichahegongheguo** [comment]
重新安装后再次启动 Hugo，报错如下：

Built in 58988 ms
Error: Error building site: failed to render pages: render of "page" failed: execute of template failed: template: posts/single.html:85:140: executing "content" at <partial "function/content.html">: error calling partial: "C:\Users\time\Documents\MEGA\个人文章\myblog\themes\LoveIt\layouts\partials\function\content.html:4:19": execute of template failed: template: partials/function/content.html:4:19: executing "partials/function/content.html" at <partial "function/ruby.html" $content>: error calling partial: partial that returns a value needs a non-zero argument.

**Comment by bearkchan** [comment]
我也遇到了这个问题，请问有解决方法吗

**Comment by sanbeichahegongheguo** [comment]
> 
> 
> 我也遇到了这个问题，请问有解决方法吗

没有解决，不用这个主题，换别的了。有的theme也有这个问题， 不过也有没有这个问题的。我就用别的了。

**Comment by bearkchan** [comment]
嗯呢，我也换别的主题了，可能是因为hugo版本问题吧

**Comment by uPagge** [comment]
This project has been abandoned. I started to develop it further

https://github.com/uPagge/uBlogger/

**Comment by bearkchan** [comment]
> This project has been abandoned. I started to develop it further
> 
> https://github.com/uPagge/uBlogger/

Thx.Maybe mention it in the old project is better.Just advice.

**Comment by pokitpeng** [comment]
+1

**Comment by CindyCyber** [comment]
试试[这篇文章](https://cloud.tencent.com/developer/article/1769218)中配置主题部分里面的，修改主题目录的那个方法，我之前也遇到了这个问题，后面解决了，具体方法我忘了。

**Comment by pokitpeng** [comment]
我通过降低hugo版本解决的
```bash
go get github.com/gohugoio/hugo@v0.69.0
```

**Comment by henryken** [comment]
It starts breaking from v0.75.0.
v0.74.3 is the last version that works with the theme.

**Comment by henryken** [comment]
I think related to the change of partials behavior
https://discourse.gohugo.io/t/error-with-partials-in-0-75-0/28184

**Comment by CindyCyber** [comment]
还有一个可能就是你的Markdown文章里面有语法错误或者是空文章。

**Comment by sanbeichahegongheguo** [comment]
> 还有一个可能就是你的Markdown文章里面有语法错误或者是空文章。

没错！事实上我发现其实是这个原因。具体到我自己是我把文章的时间设置的都不对，因为一些缘故我自己把一些文章设置为2022年。

**Comment by biningo** [comment]
出现这个错误大概率是md语法写错了，可以由一下几个方向去排错:
- 如果你用typora写md的话，语法写错了也察觉不出来(本人亲身经历,插入超链接语法写错typora还是正常显示)
- 还有一个可能就是文章头的字段拼错了
- md文档不能为空

**Comment by henryken** [comment]
I've found the issue with my project. There's one md page which has an empty body.
I added "&nbsp;" and it works now with the latest Hugo v0.83.1.

**Comment by HuangNO1** [comment]
> I've found the issue with my project. There's one md page which has an empty body.
> I added "\&nbsp;" and it works now with the latest Hugo v0.83.1.

Yes! In fact, I encounter this same problem today, and it worked after I adding "\&nbsp;" in the files' bodies which are in the directory,  "content/tags/". There are four md pages that we have to fix.

```
$ tree                     
.
├── configuration
│   └── _index.zh-cn.md
├── content
│   ├── _index.fr.md
│   └── _index.zh-cn.md
└── installation
    └── _index.zh-cn.md
```

For instance:

**before**

```md
---
title: "configuration"
---
```

**after**

```md
---
title: "configuration"
---
&nbsp;
```

**Comment by iuvhub** [comment]
> ### Describe the bug 描述你遇到的错误
> 
> 
> 
> 输入 `hugo server -D`，会出现：
> 
> 
> 
> `\content\posts\index.zh-cn.md:1:1": timed out initializing value. This is most likely a circular loop in a shortcode`
> 
> 
> 
> 这样的报错。请问是正常情况吗？
> 
> 
> 
> ### Screenshots 屏幕截图
> 
> 
> 
> ![图片](https://user-images.githubusercontent.com/32977199/109810018-f80adc80-7c63-11eb-8829-7b96c72feeac.png)
> 
> 
> 
> 
> 
> ### Build Environment 构建环境
> 
> 
> 
> - OS:  Windows 10
> 
> - Theme version/commit [0.2.10, f787a4e5]
> 
> - Hugo version [v0.62.2-83E50184]
> 
> 

你装了hugo_extended了吗

**Comment by whuwangyong** [comment]
我用最新的0.92也没问题啊。建议先把自带的一些文章删除，有些文章里面包含了连接 twitter 等shortcode，在国内是连不上的，然后报timeout。
